bin/gobblin.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Can we leave `gobblin.sh` relatively simple and instead have `gobblin-cli.sh` and `gobblin-service.sh`? `gobblin.sh` would just redirect to the correct place depending on the first argument.

Reply:
@jhsenjaliya since this is a big change, and the fact that we use the existing shell scripts in some of our internal use cases, it would be great if you can provide documentation on how to run common commands using the new script.

Reply:
yes, i have updated the doc with updated help and sample commands.